.github/workflows/windows.yml

```
@@ -622,7 +622,7 @@ jobs:
             config: >-
               -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DCURL_USE_LIBSSH2=ON
               -DCURL_USE_SCHANNEL=OFF -DCURL_USE_WOLFSSL=ON -DUSE_NGTCP2=ON
-              -DCURL_USE_GSASL=ON
+              -DCURL_USE_GSASL=ON -DUSE_HTTPSRR=ON -DUSE_ECH=ON
```

```suggestion
              -DCURL_USE_GSASL=ON
              -DUSE_HTTPSRR=ON -DUSE_ECH=ON
```
Sync formatting with BoringSSL.

---

Done.